<!DOCTYPE html>
<html lang="en">
<head>
    <meta name=generator content="Jekyll">
    <meta charset=utf-8>
    <meta http-equiv=x-ua-compatible content="IE=edge">
    <title>Christian Ponte</title>
    <meta name=viewport content="width=device-width,initial-scale=1">
    <meta property="og:title" content="Christian Ponte">
    <!-- <meta property="og:description" content> -->
    <meta property="og:type" content="website">
    <meta property="og:url" content="https://chponte.github.io">
    <meta property="og:updated_time" content="2020-10-27T11:12:25+00:00">
    <!-- <meta name=twitter:card content="summary">
    <meta name=twitter:title content="Archie">
    <meta name=twitter:description content> -->
    <link rel="stylesheet" type="text/css" href="https://chponte.github.io/assets/css/main.css">

    <!-- Google Analytics -->
    <script>
    (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
    (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
    m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');

    ga('create', 'UA-129572314-1', 'auto');
	ga('send', 'pageview', { 'page': location.pathname + location.search + location.hash});
	ga('set', 'anonymizeIp', false);
    </script>
    <!-- End Google Analytics -->
    </head>

<body>
    <div class="content">
        <header>
    <div class="main">
        <span>CHRISTIAN PONTE</span>
    </div>
    <nav>
        
        
            
        
            
                <a href="/">home</a>
            
        
            
                <a href="/publications/">publications</a>
            
        
            
                <a href="/software/">software</a>
            
        
            
                <a href="/about/">about me</a>
            
        
    </nav>
</header>

        <main class="list">
  <h1>Journal papers:</h1>
  
  <ul>
    
    <li>
      
<div class="header">
  <div class="publication-header">
    <div class="publication-type">
      
      <span>
        IEEE/ACM TCBB :: 15 October 2020
      </span>
      
    </div>
    <div class="publication-buttons">
      
      
      <i class="ai ai-open-access fa-fw"></i>
      
      
      <a href="https://pubmed.ncbi.nlm.nih.gov/33055017/"><i class="ai ai-pubmed fa-fw"></i></a>
      
      
      <a href="https://doi.org/10.1109/TCBB.2020.3030312"><i class="ai ai-doi fa-fw"></i></a>
    </div>
  </div>
  <h2><a href="https://chponte.github.io/publications/tcbb-epistasis-review.html">Evaluation of Existing Methods for High-Order Epistasis Detection</a></h2>
  <span>
    
    
    Christian Ponte-Fernández
    
    Jorge González-Domínguez
    
    Antonio Carvajal-Rodríguez
    
    María J. Martín
    
  </span>
</div>
<div class="body publication">
  <p>Finding epistatic interactions among loci when expressing a phenotype is a widely employed strategy to understand the genetic architecture of complex traits in GWAS. The abundance of methods dedicated to the same purpose, however, makes it increasingly difficult for scientists to decide which method is more suitable for their studies. This work compares the different epistasis detection methods published during the last decade in terms of runtime, detection power and type I error rate, with...</p>
</div>
    </li>
    
    <li>
      
<div class="header">
  <div class="publication-header">
    <div class="publication-type">
      
      <span>
        BMC Bioinformatics :: 9 April 2020
      </span>
      
    </div>
    <div class="publication-buttons">
      
      
      <i class="ai ai-open-access fa-fw"></i>
      
      
      <a href="https://pubmed.ncbi.nlm.nih.gov/32272874/"><i class="ai ai-pubmed fa-fw"></i></a>
      
      
      <a href="https://doi.org/10.1186/s12859-020-3456-3"><i class="ai ai-doi fa-fw"></i></a>
    </div>
  </div>
  <h2><a href="https://chponte.github.io/publications/bmc-bioinformatics-toxo.html">Toxo: a library for calculating penetrance tables of high-order epistasis models</a></h2>
  <span>
    
    
    Christian Ponte-Fernández
    
    Jorge González-Domínguez
    
    Antonio Carvajal-Rodríguez
    
    María J. Martín
    
  </span>
</div>
<div class="body publication">
  <p>Background<br> Epistasis is defined as the interaction between different genes when expressing a specific phenotype. The most common way to characterize an epistatic relationship is using a penetrance table, which contains the probability of expressing the phenotype under study given a particular allele combination. Available simulators can only create penetrance tables for well-known epistasis models involving a small number of genes and under a large number of limitations.<br> <br> Results<br> Toxo is a MATLAB library...</p>
</div>
    </li>
    
    <li>
      
<div class="header">
  <div class="publication-header">
    <div class="publication-type">
      
      <span>
        IJHPCA :: 27 May 2019
      </span>
      
    </div>
    <div class="publication-buttons">
      
      
      
      
      <a href="https://doi.org/10.1177/1094342019852128"><i class="ai ai-doi fa-fw"></i></a>
    </div>
  </div>
  <h2><a href="https://chponte.github.io/publications/ijhpca-mpi3snp.html">Fast search of third-order epistatic interactions on CPU and GPU clusters</a></h2>
  <span>
    
    
    Christian Ponte-Fernández
    
    Jorge González-Domínguez
    
    María J. Martín
    
  </span>
</div>
<div class="body publication">
  <p>Genome-Wide Association Studies (GWASs), analyses that try to find a link between a given phenotype (such as a disease) and genetic markers, have been growing in popularity in the recent years. Relations between phenotypes and genotypes are not easy to identify, as most of the phenotypes are a product of the interaction between multiple genes, a phenomenon known as epistasis. Many authors have resorted to different approaches and hardware architectures in order to mitigate the...</p>
</div>
    </li>
    
  </ul>

  <h1>Conference papers:</h1>
  
  <ul>
    
    <li>
      
<div class="header">
  <div class="publication-header">
    <div class="publication-type">
      
      <span>
        HPCS 2017 :: 17 July 2017
      </span>
      
    </div>
    <div class="publication-buttons">
      
      <a href="https://doi.org/10.1109/HPCS.2017.65"><i class="ai ai-doi fa-fw"></i></a>
    </div>
  </div>
  <h2><a href="https://chponte.github.io/publications/hpcs-omp-simd-evaluation.html">Evaluation of OpenMP SIMD Directives on Xeon Phi Coprocessors</a></h2>
  <span>
    
    
    Christian Ponte-Fernández
    
    Jorge González-Domínguez
    
    María J. Martín
    
  </span>
</div>
<div class="body publication">
  <p>Nowadays, most of the available computer architectures include vector processing units. Therefore, exploiting this type of parallelism is fundamental to achieve the highest performance on modern systems. OpenMP is a parallel programming API based on a set of compiler directives. In its latest release (version 4) explicit vector programming through Single Instruction Multiple Data (SIMD) directives was added, among other features. This paper studies the ease of use and performance of these new SIMD directives,...</p>
</div>
    </li>
    
  </ul>
</main>

        <hr>

<footer>
    Built with <a href="https://jekyllrb.com/">Jekyll</a> |
    © 2020 Christian Ponte |
    Last updated: 27 October 2020
</footer>

<link href="https://fonts.googleapis.com/css2?family=Oswald:wght@300;400;500&family=PT+Serif:ital,wght@0,400;0,700;1,400;1,700&family=Roboto:ital,wght@0,300;0,400;0,500;1,300;1,400;1,500&family=Roboto+Mono:ital,wght@0,200;0,300;0,400;1,200;1,300;1,400&display=swap" rel="stylesheet">
<link rel="stylesheet" type="text/css" href="https://use.fontawesome.com/releases/v5.13.1/css/all.css">
<link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">
    </div>
</body>

</html>